src/index/Vocabulary.cpp

```diff
@@ -278,8 +304,10 @@ template <typename S, typename C, typename I>
 auto Vocabulary<S, C, I>::getRangesForDatatypes() const
     -> ad_utility::HashMap<Datatypes, std::pair<IndexType, IndexType>> {
   ad_utility::HashMap<Datatypes, std::pair<IndexType, IndexType>> result;
-  result[Datatypes::Literal] = prefix_range("\"");
-  result[Datatypes::Iri] = prefix_range("<");
+  // TODO: Should be also consider the external vocabulary here? And where is
+  // this actually needed and is it needed at all?
+  result[Datatypes::Literal] = prefix_range_internal_vocabulary("\"");
+  result[Datatypes::Iri] = prefix_range_internal_vocabulary("<");
```

Comment:
I just checked (can you implement a usage finder in your language server based NeoVim?)
It is just used for printing those ranges for debugging, and that function (`Vocabulary::printRangesForDatatypes`) is also unused, you can remove those functions as they would give us the wrong information anyway with your changes.